### PR TITLE
fix: Updated contract addresses for ARTH Loans (Stability Pool + Trove Manager)

### PR DESCRIPTION
**Pull request by comandar21**
We recently updated our smart contracts and had to redeploy the stability pool and trove manager. Updated addresses are in our documentation.

https://docs.arth.loans/protocol/deployed-addresses

## Checklist

- [x] I have followed the [Contributing Guidelines](../CONTRIBUTING.md)
- [ ] (optional) As a contributor, my Ethereum address/ENS is:
- [ ] (optional) As a contributor, my Twitter handle is:

### CODE REVIEW

File: src/apps/arth/arth.definition.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { Register } from '~app-toolkit/decorators';
```
Comment (wpoulin) [inline src/apps/arth/arth.definition.ts:1]: We recently moved the definition to db and removed definition files, meaning this file won't be necessary and you will be able to remove it

File: src/apps/arth/index.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export { ARTH_DEFINITION, ArthAppDefinition } from './arth.definition';
```
Comment (wpoulin) [inline src/apps/arth/index.ts:1]: This file won't be necessary

**Review by wpoulin — COMMENTED** [review]
Remove the 2 files mentionned and it's a ✅
Thanks sir

File: src/apps/arth/arth.definition.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  primaryColor: '#222',
+});
+
+export class ArthAppDefinition extends AppDefinition {
+  constructor() {
+    super(ARTH_DEFINITION);
+  }
+}
+
+export default ARTH_DEFINITION;
```
Comment (wpoulin) [inline src/apps/arth/arth.definition.ts:43]: You can remove this file

File: src/apps/arth/index.ts [hunk 4]
```diff
@@ -0,0 +1,2 @@
+export { ArthAppModule } from './arth.module';
+export { ArthContractFactory } from './contracts';
```
Comment (wpoulin) [inline src/apps/arth/index.ts:2]: and this one

**Comment by wpoulin** [comment]
In the commit [552c88b](https://github.com/Zapper-fi/studio/pull/2338/commits/552c88bb1a588d5a8d328179a205cc1e2692ee22)
you removed only the decorator, not the file

**Comment by senamakel** [comment]
Hey @wpoulin So we've removed the definition files as you've mentioned :) Do let us know if this works?

**Review by wpoulin — APPROVED** [review]

**Pull request merged**